app/services/google/scrape_service.rb

```diff
@@ -11,50 +11,56 @@ def initialize(keyword_id:, search_service: Google::SearchService.new, parse_ser
     def call!
       # TODO(Goose97): update failed status once we integrate with background job
       begin
-        @keyword = Keyword.find keyword_id
+        @keyword = Keyword.find(keyword_id)
       rescue ActiveRecord::RecordNotFound
         raise_keyword_not_found
       end
 
-      result = parse_service.call result_page!
-      save_scrape_result! keyword, result
+      result = parse_service.call(result_page!)
+      save_scrape_result!(keyword, result)
     end
 
     private
 
     attr_reader :keyword_id, :keyword, :search_service, :parse_service
 
     def result_page!
-      search_service.search! keyword.content
+      search_service.search!(keyword.content)
     rescue GoogleScraperRuby::Errors::SearchError => error
-      raise_unexpected_error error
+      raise_unexpected_error(error)
```

Comment:
Technical every caught error is not unexpected 🙈 Here is can be:

```suggestion
      raise_srape_error(error)
```

It matches the custom error class name too.

Reply:
Fixed in aebdfd3c7bbb57207b42bf3b24ca2788ca6886d0